staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go

```diff
@@ -50,6 +54,10 @@ func extractTag(comments []string) []string {
 	return types.ExtractCommentTags("+", comments)[tagName]
 }
 
+func extractCustomFromTag(comments []string) []string {
```

Comment:
Use `Explicit` instead of `Custom` everywhere, including commit messages

Reply:
Done - sorry, I changed the tags only and forgot about the functions.